README.md

```diff
@@ -5,7 +5,52 @@ With two modes (FS and SF), this tool performs a sequence of fuzzing and concoli
 
 [__AFL__](http://lcamtuf.coredump.cx/afl/) is used for (blackbox) fuzzing. Ideally, this stage should cover most of the easy-to-reach functions in the programs. 
 
-[__KLEE22__](https://github.com/tum-i22/klee22/tree/sonar) is used for concolic execution. It is a custom fork of KLEE with a specialized implementation of targeted path search, called *sonar* search. Ideally, this stage should cover the (hard-to-reach) functions that were not discovered with fuzzing in the first step. Please use our [KLEE installation guide](https://github.com/tum-i22/klee-install). 
+[__KLEE22__](https://github.com/tum-i22/klee22/tree/sonar) is used for concolic execution. It is a custom fork of KLEE with a specialized implementation of targeted path search, called *sonar* search. Ideally, this stage should cover the (hard-to-reach) functions that were not discovered with fuzzing in the first step. 
+
+## Prerequisites
+Munch requires the following softwares:
+- [Macke](https://github.com/tum-i22/macke)
```

Comment:
Not sure if Munch necessarily need Macke. In shouldn't, in theory, but if it does then we need to fix that. If not now then later :)